mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+  return "Hello, my name is " + name + "and I am " + age + "years old";
```

Comment:
Attention to detail is key with every program. The test will fail because there needs to be space as below:
```
return "Hello, my name is " + name + " and I am " + age + " years old";
```

The rest of the code is perfect.